src/filter/http_context.rs

```diff
@@ -195,6 +149,23 @@ impl Filter {
         Some(res)
     }
 
+    fn get_typed_property(&self, path: &str) -> TypedProperty {
+        let tokens = tokenize_with_escaping(path, '.', '\\');
+        match self.get_property(tokens.iter().map(AsRef::as_ref).collect()) {
+            None => {
+                debug!(
+                    "[context_id: {}]: selector not found: {path}, defaulting to ``",
+                    self.context_id
+                );
+                TypedProperty::String("".to_string())
+            }
+            Some(attribute_bytes) => match self.property_mapper.typed(path, attribute_bytes) {
+                Ok(tp) => tp,
+                Err(raw) => TypedProperty::string(raw),
```

Comment:
So, if we fail to map a known property, we try to make it a `String` which in turn may fail, when there is an invalid byte sequence, which then becomes a `Bytes`. I added support for that in the `as_literal` tho unsure what one would do this that...

Reply:
Then maybe return `Result<Option<TypedProperty>, Vec<u8>>` instead of `option` and leave the decision to the caller? 

In any case, what I would do is logging error saying.. "property A could not be parsed". 

When the caller is the pattern expression, the error could be "overlooked" as `""`, while the descriptor builder can fallback to `default` field value.

Reply:
I prefer the current approach which clearly separate the two concerns... 
Now I could _not_ fold the error case in the `TypedProperty::string` function, but I am of the opinion that's the behavior we should aim for: unknown is tried as a `String`, if it isn't proper UTF-8, we fall back to string encoded bytes (see `TypedProperty::as_string` or `::as_literal` for `TypedProperty::Bytes`)